### PR TITLE
clean up data model page

### LINKED ISSUE #59: adjust data model for regular observations
and allow missing values. Also add recommendations for resampling

### PR DESCRIPTION
**Pull request by wholmgren**
lots of changes of future tense to present tense. random other fixes.

closes #59.

### CODE REVIEW

File: data_model.md [hunk 1]
```diff
@@ -250,13 +243,13 @@ of forecast parameters):
 ### Aggregates
 {: .anchor}
 
-Each Aggregate must be associated with one or more Observations. Solar
-Forecast Arbiter will calculate aggregated observation data generated
-from these Observations. Each aggregate will also provide a list of
+Each Aggregate must be associated with one or more Observations. The Solar
+Forecast Arbiter calculates aggregated observation data
+from these Observations. Each aggregate also provides a list of
```
Comment (wholmgren) [inline data_model.md]: @alorenzo175 are these sentences about aggregates and sites correct? Seems like the implementation does not actually do this.

File: data_model.md [hunk 2]
```diff
@@ -250,13 +243,13 @@ of forecast parameters):
...
 Sites associated with the Observations. These Sites will provide
 forecast providers with the modeling parameters for each system in the
-Aggregate. Aggregate forecasts will be associated directly with an
-Aggregate. Forecast providers will be required to submit a forecast of
+Aggregate. Aggregate forecasts are directly associated with an
+Aggregate. Forecast providers are required to submit a forecast of
 the aggregated quantity.
 
 Aggregates have the following required parameters:
```
Comment (wholmgren) [inline data_model.md]: @alorenzo175 should I update these parameters to match https://api.solarforecastarbiter.org/#tag/Aggregates/paths/~1aggregates~1/post ?

File: data_model.md (hunk 1 above)
Comment (alorenzo175) [reply data_model.md]: Not correct. Right now only provides list of observations from which you can get the sites.

File: data_model.md (hunk 2 above)
Comment (alorenzo175) [reply data_model.md]: yes

File: data_model.md [hunk 3]
```diff
@@ -113,19 +114,11 @@ Site. They are:
...
--   *Network* - measurement network name, e.g. SURFRAD
--   *Well-Known Text* - describes a geometric area for a Site which may be
-    physically extended, e.g. a polygon over a city for a Site that
-	describes many distributed generation PV systems.
-
-Sites that are solar power plants will require additional parameters
-that enable power forecasts. These parameters will be listed under a
-“modeling parameters” key as follows:
+Sites that are solar power plants require additional parameters that
+enable power forecasts. These parameters are listed under a“modeling
```
Comment (alorenzo175) [inline data_model.md]: fix 'a"modeling' perhaps with 'under the "modeling parameters" key'

File: data_model.md [hunk 4]
```diff
@@ -364,12 +377,12 @@ timestamp,value,quality_flag
       },
       “values”: [
         {
-          “timestamp”: “2018-11-22T12:01:48Z”,
+          “timestamp”: “2018-11-22T12:00:00Z”,
```
Comment (wholmgren) [inline data_model.md]: should I replace all of the fancy quotes with plain quotes? while I don't really care for formatting, I'm worried that someone might copy/paste the template and end up eventually submitting fancy quotes that get rejected by the api.

Comment (alorenzo175) [reply data_model.md]: Sure. It definitely does get rejected w/ fancy quotes and not with regular

**Pull request merged**